### PR TITLE
Add WebRTC ICE servers config

### PR DESCRIPTION
**Pull request by marcovidonis**
This PR adds an optional field to the Client config, where the user can specify a list of ICE Server URLs that will be passed to Pion as part of the WebRTC config when creating a new WebRTC Peer Connection.

There is already an option for this in Pion, this PR exposes it to the torrent client:
>`ICEServers` defines a slice describing servers available to be used by ICE, such as STUN and TURN servers.

### CODE REVIEW

File: webtorrent/transport.go [hunk 1]
```diff
@@ -49,11 +49,17 @@ func (me *wrappedPeerConnection) Close() error {
...
 
-func newPeerConnection(logger log.Logger) (*wrappedPeerConnection, error) {
+func newPeerConnection(logger log.Logger, iceServers []string) (*wrappedPeerConnection, error) {
 	newPeerConnectionMu.Lock()
 	defer newPeerConnectionMu.Unlock()
 	ctx, span := otel.Tracer(tracerName).Start(context.Background(), "PeerConnection")
-	pc, err := api.NewPeerConnection(config)
+
+	pcConfig := config
+	if iceServers != nil {
```
Comment (anacrolix) [inline webtorrent/transport.go:58]: Is this check necessary? Can you just have a default value in the ClientConfig, or will pion treat nil as the default anyway?

**Comment by anacrolix** [comment]
@marcovidonis just following up, do you still want to get this in?

**Comment by marcovidonis** [comment]
Sorry for the long absence @anacrolix! Yes, I plan on picking this up next week it that's alright.

File: webtorrent/transport.go (hunk 1 above)
Comment (marcovidonis) [reply webtorrent/transport.go]: This was in fact an unnecessary check. If Pion receives `nil` or an empty list of ICE Servers, it will simply ignore it.

**Comment by anacrolix** [comment]
Cheers

**Pull request merged**